e2e/handles/handles.test.ts

```diff
@@ -119,6 +119,10 @@ describe('🤝 Handles', function () {
         assert(msaOption.isSome, 'msaOption should be Some');
         const msaFromHandle = msaOption.unwrap();
         assert.equal(msaFromHandle.toString(), msa_id.toString(), 'msaFromHandle should be equal to msa_id');
+
+        // Check that the rpc returns the index as > 0
+        const apiCheck = await ExtrinsicHelper.apiPromise.rpc.handles.checkHandle(handle);
+        assert(apiCheck.suffix_index > 0);
```

Comment:
Since the returned value is a `u16`, does the `>` operator work correctly, or should you `.toNumber()` the result first?

Reply:
I tested and it was valid. JS does the coercion.

Reply:
meh. Stylistically, I prefer explicit unwrapping/conversion of SCALE to native JS types, as it makes it clear when you read the code that's what's happening. Otherwise a casual reader might assume that `suffix_index` is a regular JS `number` when it actually isn't.

But, this is my non-blocking preference, as this is a test & not deployed code.

Reply:
nvm... apparently, RPC calls return native JS; it's only runtime calls that return SCALE.